### ISSUE TITLE
change interface behavior involving duplicate patterns to round robin

### ISSUE THREAD

**Issue by Hiagg** [GTNewHorizons/GT-New-Horizons-Modpack#13956]
### Your Pack Version

2.3.4 nightly (idk which)

### Your Proposal

make it so me interfaces, dual or otherwise, round robin when there is duplicate patterns on the network. current behavior means that if you have all the items ready for a bunch of recipes, the system splits between all of them as expected. but if you for example have 16 interfaces all sharing a pattern, but enough material for only one recipe comes in at once, then it will always repeatedly output from the first interface forever. this is especially a problem for things like scaling dire autocrafting table setups for solars since they are a huge bottleneck, because there is no stocking bus to simply share 1 inventory between them all. or really any setup where someone may not want to passive the fluid but still need large parallels such as AAL. 

round robin behavior would mean that even if you trickle in items for 1 recipe at a time, itll always move on to the next interface in the chain to output. ensuring it doesnt get locked in a long craft time, such as outputting 1k recipes into 1 infinity chest at 1 aal when there was 31 others free sitting empty.

### Your Goal

it allows more flexibility in gameplay so you dont have to passive every fluid or have every machine step be able to saturate the last just to get good parallels, while also fixing a few other annoying cases

### Your Vision

this would improve things ty

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by firenoo** [comment]
Isn't this the point of p2p interfaces

**Comment by S4mpsa** [comment]
P2P Interfaces do indeed cycle between outputs, atleast they did back when I was using them.

**Comment by Hiagg** [comment]
i have come back to this issue with updated news on 2.4 using dual interface p2p, and can confirm it does not at all solve the issue and functions completely identically to the original post.

current setup is my t9 eoh block AALs all using dual interface p2p, the stabilizers which take both other blocks as input have the ingredients come in slowly, and in my world, it puts 1000 recipes in one AAL, sometimes puts recipes in a 2nd AAL, and then entirely ignores anything above 2

**Comment by Wumianzhe** [comment]
Yeah, similar situation. P2P interfaces cycle, but slowly/weirdly. They seem to put big batches of crafts started at similar time mostly in one machine
